infrastructure/aat.tfvars

```diff
@@ -3,7 +3,7 @@ node_environment = "preprod"
 additional_hostname = "sscs-cor.nonprod.platform.hmcts.net"
 idam_url = "https://idam-web-public.aat.platform.hmcts.net"
 idam_api_url = "https://idam-api.aat.platform.hmcts.net"
-tribunals_api_url = "http://sscs-tribunals-api-aat.service.core-compute-aat.internal"
+tribunals_api_url = "http://sscs-tribunals-api-pr-875.service.core-compute-preview.internal"
```

Comment:
Please update TRIBUNALS_API_URL instead of PR

Reply:
changed